types/svg-path-bounds/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for svg-path-bounds 1.0
+// Project: https://www.npmjs.com/package/svg-path-bounds
+// Definitions by: larsmunkholm <https://github.com/larsmunkholm/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = svgPathBounds;
+
+declare function svgPathBounds(path: string): [number, number, number, number];
```

Comment:
```suggestion
declare function svgPathBounds(path: string): [left: number, top: number, right: number, bottom: number];
```
this is optional, named tuples support is now wide (TS4.0):
https://www.typescriptlang.org/play?ts=4.0.2#example/named-tuples